app/templates/_package.json

```diff
@@ -4,7 +4,7 @@
   "description": "<%= moduleDesc %>",
   "main": "index.es5.js",
   "scripts": {
-    "coverage": "isparta cover _mocha index.js -- --require babel/register",
+    "coverage": "babel-node node_modules/.bin/isparta cover _mocha",
```

Comment:
It didn’t work for me. Results in `Transformation error` and `No coverage information was collected`.